components/utilities/ai.js

```diff
@@ -0,0 +1,184 @@
+/* AI logic to play a double */
+// Need to handle logic if there are 3/4 of a kind
+// This leads to different combinations of doubles.
+const double = (hand) => {
+  const dupesObject = createDuplicatesObject(hand);
+  const combinations = [];
+
+  for (const value in dupesObject) {
+    if (dupesObject[value].length === 2) {
+      combinations.push(dupesObject[value]);
+    } else if (dupesObject[value].length > 2) {
+      for (let i = 0; i < dupesObject[value].length; i++) {
+        for (let j = i + 1; j < dupesObject[value].length; j++) {
+          combinations.push([dupesObject[value][i], dupesObject[value][j]]);
+        }
+      }
+    }
+  }
+  return combinations;
+}
+/* AI logic to play a triple */
```

Comment:
for cleaner readability, add a line break between functions

```suggestion
}

/* AI logic to play a triple */
```